src/services/dumper/dumper.js

```diff
@@ -135,93 +91,82 @@ class Dumper {
       dependencies,
     };
 
-    this.writeFile(projectPath, 'package.json', `${JSON.stringify(pkg, null, 2)}\n`);
+    this.writeFile('package.json', `${JSON.stringify(pkg, null, 2)}\n`);
   }
 
   static tableToFilename(table) {
     return _.kebabCase(table);
   }
 
-  static getDatabaseUrl(config) {
+  static getDatabaseUrl(dbConfig) {
     let connectionString;
 
-    if (config.dbConnectionUrl) {
-      connectionString = config.dbConnectionUrl;
+    if (dbConfig.dbConnectionUrl) {
+      connectionString = dbConfig.dbConnectionUrl;
     } else {
-      let protocol = config.dbDialect;
-      let port = `:${config.dbPort}`;
+      let protocol = dbConfig.dbDialect;
+      let port = `:${dbConfig.dbPort}`;
       let password = '';
 
-      if (config.dbDialect === 'mongodb' && config.mongodbSrv) {
+      if (dbConfig.dbDialect === 'mongodb' && dbConfig.mongodbSrv) {
         protocol = 'mongodb+srv';
         port = '';
       }
 
-      if (config.dbPassword) {
+      if (dbConfig.dbPassword) {
         // NOTICE: Encode password string in case of special chars.
-        password = `:${encodeURIComponent(config.dbPassword)}`;
+        password = `:${encodeURIComponent(dbConfig.dbPassword)}`;
       }
 
-      connectionString = `${protocol}://${config.dbUser}${password}@${config.dbHostname}${port}/${config.dbName}`;
+      connectionString = `${protocol}://${dbConfig.dbUser}${password}@${dbConfig.dbHostname}${port}/${dbConfig.dbName}`;
     }
 
     return connectionString;
   }
 
-  static isDatabaseLocal(config) {
-    const databaseUrl = Dumper.getDatabaseUrl(config);
+  static isDatabaseLocal(dbConfig) {
```

Comment:
This could be in a utils function, and moved to the parent level as both of the dumpers will require it.

Reply:
Actually nop